### ISSUE TITLE
Attacking Gym

### ISSUE THREAD

**Issue by ghost** [FeroxRev/Pokemon-Go-Rocket-API#149]
so when i attack a gym my pokemon get hurt but dont hurt the opponents pokemon any ideas why?

**Comment by ghost** [comment]
how do you attack? it would be easier to debug if you would post some code...

**Comment by ghost** [comment]
Look at the code it's all there to start to battle and all there to set battle actions 
Just when in there it doesn't attack

**Comment by ghost** [comment]
What defendingPokemonId and/or attackingPokemonIds do you pass? are you sure all attacking pokemons are able to attack (check health).

**Comment by ghost** [comment]
Yes before sending them into battle it will revive or heal any Pokemon to max in the 6 selected

**Comment by ghost** [comment]
noob question, but did you checked the responses?

**Comment by ghost** [comment]
Yes responses all act normal and even dumped it from legit playing and same have you got email or something as don't want the code going public

**Comment by thekjohn** [comment]
http://pastebin.com/vJEj6i2r

**Comment by AresDraco** [comment]
Hum... I could not made it attack, I am stuck at starting battle, then I receive the 2 attacks and the join acction but then I'm lost, any idea how to populate the actions list with attack?

**Comment by jamie2016-1** [comment]
Im stuck at StartGymBattle I get an empty response all the time. You got this part working? I have it working under python but this API i get nothing.
